hips/hip-XXXX.md

```diff
@@ -0,0 +1,131 @@
+This is to provide better UX that simply erroring on any potential new fields in `Chart.yaml` that are unknown to the current version of Helm.
+
+`helm create` will pre-fill Chart.yaml's `minimumHelmVersion` field with the current version of Helm.
+While this may not be strictly necessary, it will provide a backstop for features that may be included in the chart by the user for their current Helm version or `helm create`.
+The user can remove/reduce the specified `minimumHelmVersion` if desired.
+
+`helm lint` will produce a warning, if the current version of Helm is newer than the minimum version constraint.
+Or the `minimumHelmVersion` field is unset for the given chart.
+This is to encourage users to update the minimum version to currently utilized version of Helm (that the user is using to develop the chart).
+
+
+## Backwards compatibility
+
+As `minimumHelmVersion` is an optional field, it will be inferred to be unset in existing charts.
+And when unset, Helm will retain existing behavior (won't implement any checks)
+
+Older versions of Helm which don't understand the `minimumHelmVersion` field will actually ignore the field even if set (as they load `Chart.yaml` ignoring unknown fields).
+However, there isn't anything that can practically be done to address this encoded behavior.
+
+
+## Security implications
+
+None
+
+
+## How to teach this
+
+Update docs for `Chart.yaml`
+
+
+## Reference implementation
+
+N/A
+
+
+## Rejected ideas
+
+**Generic version contraints:**
+It is thought a "minimum" constraint is preferred over a generic "constraint" system.
+ie. where a user could list multiple version constraints to be satisfied
+Similar to e.g. [pip's requirement-specifiers][3].
+This is because Helm makes great effort to remain [backwards compatible][4].
+And so it is presumed unnecessary that a user would need to specify anything but a single minimum version constraint.
+And in the future when Helm does make breaking changes to charts, it is expected the chart API version will be incremented to explicitly signify this new incompatiblity.
+
+**Capabilities:**
+Providing a capabilities based system.
+Rather than just directly inferring Helm capabilities from its version.
+
+A `Chart.yaml` could describe its required features ie. capabilities it expects/requires the Helm version to provide.
+From which Helm could inspect and matches against features/capabilities the in use version does provide.
+However, since core Helm capabilities are only every additive due to Helm's [backwards-compatibility rules][4], a capabilities based system is redundant.
+
+*NB: at the time of writing, plugins for extending Helm's functionality are being heavily discussed.
+And while plugins will likely allow extending capabilities independent of the Helm version.
```

Comment:
Combine with the following sentence? Its currently a fragment

Reply:
oops, right